packages/@ember/-internals/owner/index.ts

```diff
@@ -55,25 +55,24 @@ export const OWNER = symbol('OWNER');
   `audioType` passed as an attribute:
 
   ```app/components/play-audio.js
-  import Component from '@ember/component';
-  import { computed } from '@ember/object';
+  import Component from '@glimmer/component';
   import { getOwner } from '@ember/application';
 
   // Usage:
   //
-  //   {{play-audio audioType=model.audioType audioFile=model.file}}
+  //   <PlayAudio @audioType={{model.audioType}} @audioFile={{model.file}}/>
   //
-  export default Component.extend({
-    audioService: computed('audioType', function() {
+  export default class extends Component {
+    get audioService() {
       let owner = getOwner(this);
-      return owner.lookup(`service:${this.get('audioType')}`);
-    }),
+      return owner.lookup(`service:${this.args.audioType}`);
+    }
 
     click() {
```

Comment:
I think the name of this method should be changed so it doesn't give the impression that glimmer components also have DOM event handlers. What do you think of something like `onPlay`?